src/Simplify.elm

```diff
@@ -2875,8 +2875,8 @@ stringReverseChecks ({ parentRange, fnRange, firstArg } as checkInfo) =
 
 stringSliceChecks : CheckInfo -> List (Error {})
 stringSliceChecks checkInfo =
-    case ( checkInfo.firstArg, checkInfo.secondArg ) of
-        ( Node _ (Expression.Integer 0), _ ) ->
+    case ( checkInfo.firstArg, checkInfo.secondArg, checkInfo.thirdArg ) of
```

Comment:
I think nested if expressions could make more sense for readbility here. What do you think? :thinking:

Reply:
I think either option looks a bit cursed... like
```elm
if Maybe.map Node.value checkInfo.thirdArg == Just (Expression.Literal "") then
```

Reply:
Right. I think that the condition is not great (though not that bad, I'm fine with it), but if it makes the rest of the conditions easier, I think it's probably better. Feel free to leave this out if you want to, I can take a look later.